internal/brokers/broker_test.go

```diff
@@ -210,12 +210,13 @@ func TestIsAuthenticated(t *testing.T) {
 
 		cancelFirstCall bool
 	}{
-		"Successfully authenticate":                              {sessionID: "success"},
-		"Successfully authenticate after cancelling first call":  {sessionID: "IA_second_call", secondCall: true},
-		"Denies authentication when broker times out":            {sessionID: "IA_timeout"},
-		"Adds default groups even if broker did not set them":    {sessionID: "IA_info_missing_groups"},
-		"No error when auth.Next and no data":                    {sessionID: "IA_next"},
-		"No error when broker returns userinfo with empty gecos": {sessionID: "IA_info_missing_gecos"},
+		"Successfully authenticate":                                        {sessionID: "success"},
+		"Successfully authenticate after cancelling first call":            {sessionID: "IA_second_call", secondCall: true},
+		"Denies authentication when broker times out":                      {sessionID: "IA_timeout"},
+		"Adds default groups even if broker did not set them":              {sessionID: "IA_info_missing_groups"},
+		"No error when auth.Next and no data":                              {sessionID: "IA_next"},
+		"No error when broker returns userinfo with empty gecos":           {sessionID: "IA_info_missing_gecos"},
+		"No error when broker returns userinfo with group with empty GUID": {sessionID: "IA_info_missing_guid"},
```

Comment:
is it missing empty or missing from what the broker return? I think the broker return nothing which is then unmarshall to empty, but from the broker perspective, we didn’t get anything, or am I wrong? (just on naming the test case)

Reply:
This test case is to cover the case where the broker returns a group with empty ugid (now I realized that I switched around the letters :sweat_smile:) as in `{"name": "group1", "ugid": ""}`